type-o-matic/Program.cs

```diff
@@ -286,6 +286,11 @@ static void TLFunctionsForStream (Stream stm, PlatformName platform, Dictionary<
 
 		static SLAttribute AvailableAttributeForFunc (PlatformName platform, TypeDefinition type)
 		{
+
+			var map = TypeAggregator.AvailableMapForPlatform (platform);
+			string available;
+			if (map != null && map.TryGetValue (type.FullName, out available))
```

Comment:
With the new c# syntax you can remove string available and do something like:
```suggestion
			if (map != null && map.TryGetValue (type.FullName, out string available))
```